### PR TITLE
add using section to readme; add sample script

### PR DESCRIPTION
**Pull request by mkinney**
added info on how to use library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,9 @@
 This is a simple python interface of RARBG.to.
 
 The api object will automatically fetch/refresh the token and take care rate control (1 request/2 seconds)
+
+# using
```
Comment (verybada) [inline README.md:9]: Using `Installation` as session name would be nice.

File: README.md [hunk 2]
```diff
@@ -5,3 +5,9 @@
...
+
+    pip install RarbgAPI
+
+For an example, see matrix.py
```
Comment (verybada) [inline README.md:13]: And we can have a  `quickstart` session to show a way to go instead of submit a new file.

**Review by verybada — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mkinney) [reply README.md:9]: I don't understand. Can you elaborate?

File: README.md (hunk 2 above)
Comment (mkinney) [reply README.md:13]: I don't understand. Can you elaborate?

**Comment by verybada** [comment]
I have added a quickstart

**Pull request closed without merging**